### PR TITLE
Adds resistor-color exercise

### PR DESCRIPTION
**Pull request by amscotti**
Hi, this PR is to add a new exercise for `resistor-color`, [here is a link to the problem-specifications](https://github.com/exercism/problem-specifications/tree/master/exercises/resistor-color)

I'm still learning Dart, so feel free to point out anything that needs to be change or improved. I'm also happy to change the `config.json` to better fit your layout for the track.

Thanks! :)

### CODE REVIEW

File: exercises/resistor-color/lib/example.dart [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class ResistorColor {
+  final colors = ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"];
```
Comment (jvarness) [inline exercises/resistor-color/lib/example.dart]: You might consider moving this to the package level and making it private.

Comment (Stargator) [reply exercises/resistor-color/lib/example.dart]: One or the other works for me. Or both. :)

Comment (Stargator) [reply exercises/resistor-color/lib/example.dart]: @jvarness It looks like it needs to be public in order to fit pass a test.

**Comment by jvarness** [comment]
I agree with your comments @Stargator.

@amscotti I merged your other branch so this might have a conflict now. Once you've rebased one of us will merge it.

**Comment by amscotti** [comment]
@jvarness All rebased, thanks! :)

**Pull request merged**